### DISCUSSION TITLE
Changing .footnotes font-size

### DISCUSSION THREAD

**Discussion by ImaCrea** [thuliteio/doks#577] [Q&A] [answered]
Hello there,

I'm trying to change the font-size of .footnotes class. I'm on the child theme and I followed what's explained on other posts here, which is : copying `_global.scss` from `.node_modules/@hyas/doks/assets/scss/common` to `assets/scss/common` at the root of my project.

Then I updated the newly copied `_global.scss` by adding at the end `.footnotes {font-size:80%;}` but it doesn't seem to change anything.

Thanks for your help.

**Comment by h-enk** [comment, marked as answer]
Try clearing the cache: `docker builder prune` / `npm run clean`?

Could you share a link to the repo?

**Reply by ImaCrea** [reply]
Hello, I tried your suggestion. Still blocked.
I can't really give access to that repo I'm afraid. 

What should I look at please?

**Reply by h-enk** [reply]
Yeah, I can confirm that the overwrite does not work, dunno why though (looks like a Hugo thing).

What you can do:
- copy `./node_modules/@hyas/doks/assets/scss/app.scss` to `./assets/scss/app.scss`
- add `@import "common/custom";` (line 17)
- create `./assets/scss/common/_custom.scss` + add your custom scss

**Reply by ImaCrea** [reply]
Hello, thanks for your help. It worked ! ✌️
